sources/packages/workflow/BPMN/camunda-8/load-assessment-consolidated-data.bpmn

```diff
@@ -70,24 +70,24 @@
-          <zeebe:header key="partnerPermanentDisabilityBenefits" value="$.supportingUsers.Partner1.supportingdata.partnerTotalPDBenefits" />
-          <zeebe:header key="partnerStudentStudyWeeks" value="$.supportingUsers.Partner1.supportingdata.partnerStudentStudyWeeks" />
-          <zeebe:header key="studentCRAReportedIncome" value="$.student.craReportedIncome" />
+          <zeebe:header key="parent2TotalIncome" value="$.supportingUsers.Parent2.supportingdata.totalIncome" />
+          <zeebe:header key="partner1SupportingUserId" value="$.supportingUsers.Partner1.id" />
+          <zeebe:header key="studentDataLivingWithPartner" value="$.supportingUsers.Partner1.supportingdata.partnerLivingWithStudent" />
+          <zeebe:header key="partner1ChildSpousalSupportCost" value="$.supportingUsers.Partner1.supportingdata.partnerChildSpousalSupportCost" />
+          <zeebe:header key="partner1DisabilityAssistance" value="$.supportingUsers.Partner1.supportingdata.partnerTotalIncomeAssistance" />
+          <zeebe:header key="partner1EmploymentInsuranceBenefits" value="$.supportingUsers.Partner1.supportingdata.partnerTotalEmploymentInsurance" />
+          <zeebe:header key="partner1TotalStudentLoan" value="$.supportingUsers.Partner1.supportingdata.partnerTotalStudentLoan" />
+          <zeebe:header key="partner1PermanentDisabilityBenefits" value="$.supportingUsers.Partner1.supportingdata.partnerTotalPDBenefits" />
+          <zeebe:header key="partner1StudentStudyWeeks" value="$.supportingUsers.Partner1.supportingdata.partnerStudentStudyWeeks" />
+          <zeebe:header key="studentDataCRAReportedIncome" value="$.student.craReportedIncome" />
           <zeebe:header key="studentTaxYear" value="$.student.taxYear" />
           <zeebe:header key="parent1CRAReportedIncome" value="$.supportingUsers.Parent1.craReportedIncome" />
           <zeebe:header key="parent2CRAReportedIncome" value="$.supportingUsers.Parent2.craReportedIncome" />
-          <zeebe:header key="partnerCRAReportedIncome" value="$.supportingUsers.Partner1.craReportedIncome" />
-          <zeebe:header key="studentIncomeAppealData" value="$.appeals.studentIncomeAppeal.submittedData" />
-          <zeebe:header key="partnerIncomeAppealData" value="$.appeals.partnerIncomeAppeal.submittedData" />
-          <zeebe:header key="courseLoad" value="$.offering.courseLoad" />
-          <zeebe:header key="weeks" value="$.offering.studyBreaks.totalFundedWeeks" />
+          <zeebe:header key="partner1CRAReportedIncome" value="$.supportingUsers.Partner1.craReportedIncome" />
+          <zeebe:header key="appealsStudentIncomeAppealData" value="$.appeals.studentIncomeAppeal.submittedData" />
+          <zeebe:header key="appealsPartnerIncomeAppealData" value="$.appeals.partnerIncomeAppeal.submittedData" />
+          <zeebe:header key="offeringCourseLoad" value="$.offering.courseLoad" />
+          <zeebe:header key="offeringWeeks" value="$.offering.studyBreaks.totalFundedWeeks" />
```

Comment:
are we converting all the dmn headers also from weeks to offeringWeeks? I think we are referring everywhere as weeks.

Reply:
Yes, @abschwenker is taking care of this.